src/features/dynamicmenu/DragButton.gd

```diff
@@ -0,0 +1,30 @@
+extends Button
+
+
+var drop_data: Variant = self
```

Comment:
Dragbutton is specifically implemented to be versatile (I did this on purpose). The reason for this is that it is a class that is meant to be extended and `drop_data` is meant to be set by the child class. The reason I made it a variant is because `data` in the `_can_drop_data` [function is already defined as a variant](https://docs.godotengine.org/en/stable/classes/class_control.html#class-control-method-drop-data).

Maybe defining it as `self` is the mistake here instead? That way the user *has* to extend the class for it to work.